backend/src/main/java/com/bakdata/conquery/models/config/auth/MultiInstance.java

```diff
@@ -0,0 +1,12 @@
+package com.bakdata.conquery.models.config.auth;
+
+import javax.validation.constraints.NotEmpty;
+
+/**
+ * Allows a {@link com.bakdata.conquery.models.config.PluginConfig} to be defined multiple times with different ids.
```

Comment:
hä

Reply:
Ich hatte mir überlegt, dass man manche Plugins mehrere Male definieren kann in der Config, wie z.B. FormBackend, manche aber nur einmal, wie zum Beispiel ManualPlugin